h/static/styles/excerpt.scss

```diff
@@ -5,6 +5,10 @@
   .excerpt {
     transition: max-height $expand-duration ease-in;
     overflow: hidden;
+
+    p {
```

Comment:
This styling is related to the excerpt's content rather than the excerpt, so it should live alongside the classes that are used to style the annotation content. In this case the `<markdown>` component unfortunately uses the generic `.styled-text` class which is in turn used in a bunch of other places.

One way to solve this would be to make `.styled-text` a mixin so that it can be re-used in different classes, create a `.styled-text` class which just imports that mixin so that other places which use that class today are unaffected and then replace the styling for the markdown body with a different class which imports that mixin and overrides the styling of paragraphs specifically. eg. https://gist.github.com/robertknight/a7ba647b95f6c30f44be9c3aac3810f8

The overriding goal here is to make it as easy as possible to find all the styling which affects a specific element.